docs/reference/troubleshooting/data/increase-cluster-shard-limit.asciidoc

```diff
@@ -0,0 +1,20 @@
+[[increase-cluster-shard-limit]]
+== Maximum number of collocated primaries and replicas per node has been reached 
```

Comment:
This title is a bit messy. Any better suggestions?

Reply:
"Maximum number of shards per node has been reached" ?